botsquare/botsquare.ino

```diff
@@ -1,35 +1,28 @@
 #include <Wire.h>
 #include <Adafruit_MotorShield.h>
-#include "utility/Adafruit_PWMServoDriver.h"
```

Comment:
Was this library deprecated, unused by this code,  subsumed by one of the header files above?